### PR TITLE
added support for selecting items for a select box

### PR DESCRIPTION
**Pull request by dereke**

### CODE REVIEW

File: test/browserMonkeySpec.js [hunk 1]
```diff
@@ -93,6 +93,21 @@ describe('browser-monkey', function () {
...
+    var selected = false;
+
+    eventuallyInsertHtml(
+      $('<select class="element"><option>First</option><option>Second</option></select>').change(function (e) {
+        selected = true;
+      })
+    );
+
+    return promise.then(function () {
+      expect(selected).to.equal(true);
```
Comment (refractalize) [inline test/browserMonkeySpec.js]: How do you know which one was selected?

Comment (dereke) [inline test/browserMonkeySpec.js]: Ah!
You are right it only checks that something changed.
I'll fix that

**Comment by refractalize** [comment]
Would be good to be able to assert the current selection, e.g. `.shouldHave({selected: 'Second'})` or `.shouldHave({selectedValue: 'second'})` in the case of `<option value="second">Second</option>`

**Comment by refractalize** [comment]
thanks! this is in v1.11.0

**Pull request merged**